tests/validation/tests/v3_api/test_certificate.py

```diff
@@ -0,0 +1,571 @@
+        )
+
+    def test_certificate_create_validcert_for_all_ns(self):
+        """
+        Test steps:
+        1. Validate the workload available in ns-certificate namespace
+        2. Create an ingress including trusted certificate scoped for all
+        namespace and route53 host.
+        3. validate the ingress using endpoint
+        """
+        ingress_name = random_test_name("ingress-test")
+        host = route_entry_53_1
+        path = "/name.html"
+        rule = {"host": host,
+                "paths": [{"path": path, "workloadIds": [self.workload.id],
+                           "targetPort": "80"}]
+                }
+        tls = {"certificateId": self.certificate_all_ns_valid.id,
+               "hosts": [host]
+               }
+        validate_workload(self.p_client, self.workload, "deployment",
+                          self.ns.name)
+        self.ingress = self.p_client.create_ingress(
+            name=ingress_name, namespaceId=self.ns.id, rules=[rule], tls=[tls]
+        )
+        wait_for_ingress_to_active(self.p_client, self.ingress)
+        validate_ingress_using_endpoint(
+            self.p_client, self.ingress, [self.workload], certcheck=True,
+            is_insecure=True
+        )
+
+    def test_certificate_create_validcert_for_all_ns_2(self):
+        """
+        Test steps:
+        1. Create a namespace
+        2. Create a workload in namespace created above.
+        3. Validate the workload.
+        4. Create an ingress including trusted certificate scoped for all
+        namespace and route53 host.
+        5. validate the ingress using endpoint
+        """
+        wl_name = random_test_name("workload-test")
+        wl_con = [{"name": "wk2-test",
+                   "image": TEST_IMAGE}]
+        scheduling = {"node": {"nodeId": self.node_id}}
+        ns_2 = create_ns(self.c_client, self.cluster, self.project)
+        self.workload_2 = self.p_client.create_workload(
+            name=wl_name, containers=wl_con, namespaceId=ns_2.id,
+            scheduling=scheduling
+            )
+        validate_workload(self.p_client, self.workload_2, "deployment",
+                          ns_2.name)
+        ingress_name = random_test_name("ingress-test")
+        host = route_entry_53_1
+        path = "/name.html"
+        rule = {"host": host,
+                "paths": [{"path": path, "workloadIds": [self.workload_2.id],
+                           "targetPort": "80"}]
+                }
+        tls = {"certificateId": self.certificate_all_ns_valid.id,
+               "hosts": [host]
+               }
+        self.ingress = self.p_client.create_ingress(
+            name="{}-2".format(ingress_name), namespaceId=ns_2.id,
+            rules=[rule], tls=[tls]
+        )
+        wait_for_ingress_to_active(self.p_client, self.ingress)
+        validate_ingress_using_endpoint(
+            self.p_client, self.ingress, [self.workload_2], certcheck=True,
+            is_insecure=True
+        )
```

Comment:
Can you delete the resource created in this method at the end? You can use the fixture `remove_resource`

Reply:
These are getting deleted in tear off of each tests.